src/js/components/PageHeader/PageHeader.js

```diff
@@ -0,0 +1,76 @@
+import React, { forwardRef, useContext } from 'react';
+import { ThemeContext } from 'styled-components';
+
+import { Box } from '../Box';
+import { Header } from '../Header';
+import { Heading } from '../Heading';
+import { Grid } from '../Grid';
+import { Paragraph } from '../Paragraph';
+import { ResponsiveContext } from '../../contexts/ResponsiveContext';
+
+const defaultAreas = ['actions', 'context', 'subtitle', 'title', 'empty'];
+
+const getCustomAreas = (theme) => {
+  const customAreas = [];
+  theme.pageHeader.areas.forEach((area) => {
+    if (Array.isArray(area)) {
+      area.forEach((s) => !defaultAreas.includes(s) && customAreas.push(s));
+    } else if (!defaultAreas.includes(area)) customAreas.push(area);
+  });
+
+  return customAreas;
+};
+
+const PageHeader = forwardRef(
+  ({ title, subtitle, actions, context, ...rest }, ref) => {
+    const theme = useContext(ThemeContext);
+    const size = useContext(ResponsiveContext);
+
+    // Allow caller to add their own blank areas to the Grid
+    // See: Custom story
+    const customAreas = getCustomAreas(theme);
```

Comment:
left a related comment, should we allow for areas to vary by breakpoint?
```suggestion
    const customAreas = getCustomAreas(theme, size);
```

Reply:
Done.